backend/server.ts

```diff
@@ -19,7 +19,7 @@ app.use(bodyParser.json());
 app.set('trust proxy', true);
 app.use(router);
 
-app.use((req, res, next) => {
+app.use((_req, res, next) => {
```

Comment:
Why don't we just stick with `req`?

Reply:
[this comment](https://github.com/Vonage/vonage-video-react-app/pull/143/files#r2050701537) explains it 🙏 
I'll resolve this comment and those that are similar to this one so we can keep the conversation going on that thread if needed

Reply:
That comment gives a lot of insight.  Thanks, man 🙏 

However, in this instance, can't we use the following:

```
app.use((_, res, next) => {
```

Or, even better:

```
app.use((, res, next) => {
```

Reply:
we can certainly do just `(_, res, next)` - the lint rule I added will have to stay so this is just a renaming preference. though I think if not a blocker for you, I'd like to keep it as `_req` since it's more explanatory in case we/someone else want to use this parameter in the future. 
doing `(, res, next)` won't work though because that is invalid syntax in TypeScript since parameters cannot be skipped

Reply:
Understood.  

I prefer `_` since it's a "don't care".  Giving it a name introduces unnecessary cognitive load, IMO.